docs/IMAGEUPLOADER.md

```diff
@@ -45,3 +45,13 @@ Replace `XXXXXXXXXXXXXXX` with your API key from s-ul.eu. It can be found on [yo
 |Extra headers||
 |Image link|`{url}`|
 |Deletion link|`https://s-ul.eu/delete.php?file={filename}&key=XXXXXXXXXXXXXXX`|
+
+## i.alazymeme.com
```

Comment:
There should be `###` instead of `##` to keep consistency with previous examples for other hosts